src/www/ui/async/AjaxShowJobs.php

```diff
@@ -291,6 +291,9 @@ protected function getShowJobsForEachJob($jobData)
           case 'spdx2tv':
             $jobArr['jobQueue'][$key]['download'] = "SPDX2 tag/value report";
             break;
+          case 'spdx2csv':
+            $varJobQueueRow['download'] = "SPDX2 CSV report";
+            break;
```

Comment:
Looks like the changes are breaking job view:
```
Uncaught TypeError: jobQueueData.download is undefined
    fillTableWithJobInfo http://localhost/repo/?mod=showjobs:477
    each jQuery
    fillTableWithJobInfo http://localhost/repo/?mod=showjobs:465
    createShowJobsTable http://localhost/repo/?mod=showjobs:416
    success http://localhost/repo/?mod=showjobs:547
    jQuery 4
repo:477:16
    fillTableWithJobInfo http://localhost/repo/?mod=showjobs:477
    each jQuery
    fillTableWithJobInfo http://localhost/repo/?mod=showjobs:465
    createShowJobsTable http://localhost/repo/?mod=showjobs:416
    success http://localhost/repo/?mod=showjobs:547
    jQuery 4
```

Changing it to `spdx2tv` format seems to fix the issue.

```suggestion
          case 'spdx2csv':
            $jobArr['jobQueue'][$key]['download'] = "SPDX2 CSV report";
            break;
```

Reply:
Oh, strange we didn't run into that specific problem